sets_dbs/dp_infos/copy/sqlite_to_mysql.py

```
@@ -0,0 +1,85 @@
+import pandas as pd
+from sqlalchemy import create_engine
+from tqdm import tqdm
+from pathlib import Path
+
+home_dir = os.getenv("HOME")
+project = home_dir if home_dir else "I:/ncc"
+Dir = Path(project) / "ncc_data/sets_dbs/dp_infos/copy"
+
+try:
+    path_db_1 = Dir / "fs_infos_duplict1.sqlite"
+except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)